tasks/ingress-nginx.yml

```diff
@@ -0,0 +1,7 @@
+- name: Copy ingress-nginx files to first server
+  ansible.builtin.template:
+    src: "templates/ingress-nginx-config.yml.j2"
+    dest: "{{ rke2_data_path }}/server/manifests/rke2-ingress-nginx-config.yaml"
```

Comment:
Make sure the destination directory exist. It seems like it is not. See https://github.com/lablabs/ansible-role-rke2/actions/runs/9078504653/job/25340508968

Reply:
Done